dashboard/app/models/script.rb

```diff
@@ -425,6 +426,10 @@ def self.add_script(options, data)
             name: stage_name,
             script: script
           )
+
+        stage.assign_attributes(flex: stage_flex_category)
```

Comment:
Good question!  The is a small optimization used a few places in `rake seed:scripts` to avoid unnecessary updates to the database if the model data hasn't actually changed.  I'm not sure it's needed, but following the existing pattern.